astropy/utils/iers/tests/test_iers.py

```diff
@@ -325,3 +328,28 @@ def test_IERS_B_parameters_loading_into_IERS_Auto():
             A[name][ok_A], B[name][i_B], rtol=1e-15,
             err_msg=("Bug #9206 IERS B parameter {} not copied over "
                      "correctly to IERS Auto".format(name)))
+
+
+@pytest.mark.parametrize('iersa_url', [iers.IERS_A_URL, iers.IERS_A_URL_MIRROR])
+@pytest.mark.xfail('TRAVIS')
```

Comment:
This needs to be set up such that it `xfail` only for `iers.IERS_A_URL` (the FTP service) and not the HTTP mirror. I can think of two ways:

1. Move the `xfail` to a `if` block inside the test function. But I don't think `xfail_strict` would catch it if it starts to `xpass` again some time in the future.
2. Turn this into a test class. Make the common code some hidden method that `pytest` won't collect. Then write two test methods wrappers around it, one with `xfail` marker and one without.